markdown/extensions/tables.py

```diff
@@ -85,7 +85,39 @@ def _split_row(self, row, border):
                 row = row[1:]
             if row.endswith('|'):
                 row = row[:-1]
-        return row.split('|')
+        return self._split(row, '|')
+
+    def _split(self, row, marker):
+        """ split a row of text with some code into a list of cells. """
+        if self._row_has_code(row):
+            ## fallback on old behaviour
+            return row.split(marker)
+        escaped = False
+        elements = []
+        current = ''
+        for i in row:
+            if i == '`' and (current == '' or current[-1] != '\\'):
+                ## Found an un-escaped backtick
+                escaped = not escaped
+
+            if i == marker and not escaped:
+                elements.append(current)
+                current = ''
+            else: 
+                current += i
+        if escaped:
+            ## This should never happen
+            return row.split(marker)
```

Comment:
According to the [coverage report](https://coveralls.io/builds/3919721/source?filename=markdown%2Fextensions%2Ftables.py), this is also never tested. Given the comment that this should never happen, then could you add a comment telling coverage to skip this test? -- ` # pragma: no cover`